libs/formik-elements/src/FormContainer.tsx

```diff
@@ -20,9 +20,26 @@ import * as React from "react";
 import { Formik, FormikConfig, FormikHelpers } from "formik";
 
 import { createNamedContext, ValidationError } from "@tiller-ds/util";
+import ScrollToError from "./ScrollToError";
 
 type FormikProps<V = any> = {
+  /**
+   * Specifies whether validation should occur after form submission.
```

Comment:
"after form submission" or what? Isn't validation after form submission always happening?

Reply:
No, default behavior is onBlur of a component.

Reply:
please add that to the prop description